test/packages/aws/docs/README.md

```diff
@@ -1311,10 +1311,14 @@ An example event for `ec2` looks as following:
 | aws.ec2.cpu.total.pct | The percentage of allocated EC2 compute units that are currently in use on the instance. | scaled_float |
 | aws.ec2.diskio.read.bytes | Bytes read from all instance store volumes available to the instance. | long |
 | aws.ec2.diskio.read.bytes_per_sec | Bytes read per second from all instance store volumes available to the instance. | long |
+| aws.ec2.diskio.read.count | The number of disk IO reads | long |
```

Comment:
Thanks for adding the fields!!!

Reply:
Oh I just realized this is added in `elastic-package`. How can we sync `elastic/integrations/packages/aws` with `elastic-package/test.packages/aws`?

Reply:
The goal of these packages in the elastic-package repo is to have a playground to test new feature before they reach out to Integrations. Syncing is actually the other way round - once we apply similar changes in the Integrations, I will copy over it from Integrations to the elastic-package :)

Reply:
👍 Sounds good!